src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);;
+  gap: 48px;
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns:  repeat(2, 1fr);
+  }
+
+  @media (max-width: 488px) {
+    grid-template-columns: 1fr;
+  }
+
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
+    justify-self: center;
     width: 200px;
```

Comment:
you don't need width, as it is restricted by grid column width 😉

Reply:
Fixed!